src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine+AdjustIsolatedMarginInput.kt

```diff
@@ -0,0 +1,146 @@
+package exchange.dydx.abacus.state.model
+
+import exchange.dydx.abacus.calculator.AdjustIsolatedMarginInputCalculator
+import exchange.dydx.abacus.output.input.IsolatedMarginAdjustmentType
+import exchange.dydx.abacus.responses.ParsingError
+import exchange.dydx.abacus.responses.StateResponse
+import exchange.dydx.abacus.state.changes.Changes
+import exchange.dydx.abacus.state.changes.StateChanges
+import exchange.dydx.abacus.utils.NUM_PARENT_SUBACCOUNTS
+import exchange.dydx.abacus.utils.mutable
+import exchange.dydx.abacus.utils.mutableMapOf
+import exchange.dydx.abacus.utils.safeSet
+import kollections.JsExport
+import kollections.iListOf
+import kotlinx.serialization.Serializable
+
+@JsExport
+@Serializable
+enum class AdjustIsolatedMarginInputField(val rawValue: String) {
```

Comment:
This is purely used by clients right? so can use built-in conversion? i guess you'd run into the enum naming rule (maybe disable?)